code-style/php/README.md

```diff
@@ -0,0 +1,169 @@
+# PHP Coding Standards
+![](php-logo.png)
+
+Despite still being one of the most used languages to build web applications, PHP doesn't have the best reputation. There are a number of frameworks we use that are built with PHP. In order to avoid PHP's reputation of messy codebases, we should utilize [PHP_CodeSniffer](phpcs) to detect violations of a defined coding standards.
+
+### What Standard Should I use?
+
+Many frameworks have their own standards declared. We should use the framework's defined standard when possible. For instance, when developing a WordPress theme, you should utilize the [WordPress Standard](wp). If there is not a standard defined, or you are not using a framework, you should use the [PSR-2 Standard](psr-2).  Some standards are:
+
+- [PSR-2 / Default](psr-2)
+- [WordPress](wp)
```

Comment:
```suggestion
- [WordPress][wp]
```
A subtle and confusing thing about markdown links: When using [reference-style links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links), both parts need to be in brackets instead of parens. Otherwise, it will treat it as a relative link and result in a `404`, which you can see in [the markdown preview](https://github.com/sparkbox/standard/blob/93420a4a11383c9a1bd812cea3451488a64c0863/code-style/php/README.md)

I think this change will need to apply to all of the reference links on the page.

Reply:
Great Catch @jonoliver !!! I'll make sure to fix all of those